### PR TITLE
Changes on detumbling files

### PR DESCRIPTION
**Pull request by PetrerW**
deleted isInitialised flag. Added description and units to members of DetumblingParameters structure.

---
This change is [Reviewable](https://reviewable.io/reviews/pw-sat2/pwsat2obc/96)

### CODE REVIEW

File: libs/adcs_experiments/Include/adcs_experiments/detumbling.hpp [hunk 1]
```diff
@@ -39,36 +39,39 @@ final
...
             /** @brief (Draft)   */
-            float dt = DefaultDt;
+            float dt = DefaultDt; // iteration time step, [s]
             /** @brief (Draft)   */
-            float wCutOff = DefaultWCutOff;
+            float wCutOff = DefaultWCutOff; // high-pass filter cut off frequency, [rad/s]
             /** @brief (Draft)   */
-            float bDotGain = DefaultBDotGain;
+            float bDotGain = DefaultBDotGain; // B-dot gain, [kg m^2 / s] (contant k used to calculate commanded magnetic dipole) 
+											  //s.33 "Dropbox\PW-Sat2 udostepnione\documentation\documentation_phase_c\PW-Sat2-C-01.00-ADCS-CDR.pdf"
```
Comment (pwec) [inline libs/adcs_experiments/Include/adcs_experiments/detumbling.hpp]: remove this part of comment

Comment (PetrerW) [inline libs/adcs_experiments/Include/adcs_experiments/detumbling.hpp]: Done

**Comment by ggajoch** [comment]
Review status: 0 of 2 files reviewed at latest revision, 3 unresolved discussions, some commit checks failed.

---

*[libs/adcs_experiments/detumbling.cpp, line 34 at r2](https://reviewable.io:443/reviews/pw-sat2/pwsat2obc/96#-KdzAX8LsUDN1UrQ5TvQ:-KdzAX8LsUDN1UrQ5TvR:bn2wsz8) ([raw file](https://github.com/pw-sat2/pwsat2obc/blob/1b8740bbad2c75a772e8bacbae6d6819392c830c/libs/adcs_experiments/detumbling.cpp#L34)):*
> ```cpp
>  %
>  %   globals       :
>  %       DetumblingConst.dt       - [1x1], iteration time step, [s]
> ```

Isn't it called DetumblingParameters right now?

---

*[libs/adcs_experiments/Include/adcs_experiments/detumbling.hpp, line 56 at r2](https://reviewable.io:443/reviews/pw-sat2/pwsat2obc/96#-KdzAgIKuRYvKpAeV4Fc:-KdzAgIKuRYvKpAeV4Fd:b-iiyyv3) ([raw file](https://github.com/pw-sat2/pwsat2obc/blob/1b8740bbad2c75a772e8bacbae6d6819392c830c/libs/adcs_experiments/Include/adcs_experiments/detumbling.hpp#L56)):*
> ```cpp
> 
>         /** @brief (Draft)   */
>         struct DetumblingParameters //Parameters changeable by a telecommand from Earth. Those are constants used in detumbling.
> ```

place this comment in doxygen structure

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/pw-sat2/pwsat2obc/96)*

**Comment by PetrerW** [comment]
`%
%   globals       :
%       DetumblingConst.dt       - [1x1], iteration time step, [s]`

> Isn't it called DetumblingParameters right now?

@ggajoch Yes, 'DetumblingParameters' stands for 'DetumblingConst', but important is, that members of 'DetumblingParameters' are changeable by a telecommand from earth and members of 'DetumblingState' are not.

**Pull request merged**